src/helpers/config.ts

```diff
@@ -6,9 +6,11 @@ export type CoinMetadata = {
   Description: string;
   value: string;
   Icon: CryptocurrencyIconProps['Name'];
+  ApiKeyProvider?: string;
+  isTSSSupported?: boolean;
```

Comment:
nit: isTssSupported. Also shouldn't isTssSupported be added to polygon as well?

Reply:
I was seeing only legacy support added for polygon in WRW till now, we could add it but needs someone to test it. Recoveries should work with legacy also